### PR TITLE
feat: add webpack configuration

### PR DESCRIPTION
**Pull request by rossreicks**
## Add Webpack Configuration

Pulled out webpack configuration from example project to here. Wrote readme on how to use.

Adds Feature or Fixes: # TZ-3277

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] The test workflow is passing locally

### CODE REVIEW

File: packages/webpack-config/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Extend configuration in `webpack.config.js`
+
+
+```js
+// First build your handlers object (name to path map)
+const handlers = {
+    hello: './src/handlers/hello.ts'
+}
+
+module.exports = require('@tractorzoom/webpack-config')(handlers)
```
Comment (cody-hoffman) [inline packages/webpack-config/README.md:24]: 💯

File: packages/webpack-config/index.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        mode: process.env.ENV === 'Test' || 'dev' ? 'development' : 'production',
+        module: { rules: [{ loader: 'ts-loader', test: /\.ts$/ }] },
+        output: {
+            filename: '[name].js',
+            libraryTarget: 'commonjs2',
+            path: resolve(__dirname, 'build'),
+        },
+        resolve: {
+            alias: {
+                '@src': resolve(__dirname, 'src/'),
```
Comment (cody-hoffman) [inline packages/webpack-config/index.js:15]: resolvers setup out of the box will be nice too! I don't care but I'm curious if anyone has an opinion on naming for this? @twebber90 @ahalliertz

**Review by cody-hoffman — APPROVED** [review]
LGTM

File: packages/webpack-config/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        "shared configuration",
+        "webpack"
+    ],
+    "publishConfig": {
+        "access": "public"
+    },
+    "main": "index.js",
+    "scripts": {},
+    "dependencies": {
+        "rimraf": "3.0.2",
```
Comment (twebber90) [inline packages/webpack-config/package.json:28]: ❓ should the .npmrc file have `save-exact=true`?

File: packages/webpack-config/index.js (hunk 2 above)
Comment (twebber90) [reply packages/webpack-config/index.js:15]: Haven't had to use it yet but reviewing the PRs with `@src/some/path` was easy to review 👍

**Review by twebber90 — APPROVED** [review]
👍  good docs in the readme

**Pull request merged**